queue/python/queue.py

```diff
@@ -0,0 +1,44 @@
+class Node:
+    def __init__(self, data):
+       self.data = data
+       self.next = None
+ 
+class Queue:
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp0ix2dxoe/queue/python/queue.py
+++ b/tmp/tmp0ix2dxoe/queue/python/queue.py
@@ -2,7 +2,7 @@
     def __init__(self, data):
        self.data = data
        self.next = None
- 
+
 class Queue:
     def __init__(self):
         self.head = None
```